koku/subs/trino_sql/azure/subs_summary.sql

```diff
@@ -46,13 +46,13 @@ WHERE
     AND json_extract_scalar(lower(additionalinfo), '$.vcpus') IS NOT NULL
     AND json_extract_scalar(lower(lower(tags)), '$.com_redhat_rhel') IS NOT NULL
     -- ensure there is usage
-    AND ceil(coalesce(nullif(quantity, 0), usagequantity)) > 0
+    AND ceil(coalesce(nullif(quantity, 0), 0)) > 0
```

Comment:
I'm really not sure what this condition is. We can probably reduce it to:
```suggestion
    AND quantity > 0
```